examples/flutter_gallery/lib/gallery/backdrop.dart

```diff
@@ -245,7 +254,12 @@ class _BackdropState extends State<Backdrop> with SingleTickerProviderStateMixin
 
   Widget _buildStack(BuildContext context, BoxConstraints constraints) {
     final Animation<RelativeRect> frontRelativeRect = new RelativeRectTween(
-      begin: new RelativeRect.fromLTRB(0.0, constraints.biggest.height - _kFrontClosedHeight, 0.0, 0.0),
+      begin: new RelativeRect.fromLTRB(
```

Comment:
I think this was OK, as it was on one line

Reply:
done